CONTRIBUTING.md

```
@@ -0,0 +1,168 @@
+
+
+## Label Issues
+
+One of the most important tasks in handling issues is labelling them usefully and accurately. All other tasks involving issues ultimately rely on the issue being classified so that relevant parties looking to do their tasks can find them quickly and easily.
+
+To label issues, [open up the list of unlabeled issues](https://github.com/canopas/group-track-android/issues?q=is%3Aopen+is%3Aissue+no%3Alabel+) and, **from newest to oldest**, read through each one and apply issue labels according to the table below. If you're unsure about what label to apply, skip the issue and try the next one: don't feel obligated to label every issue yourself!
+
+Label | Apply When | Notes
+--- | --- | ---
+`bug` | Cases where the code (or documentation) is behaving in a way it wasn't intended to. | If something is happening that surprises the *user* but does not go against the way the code is designed, it should use the `enhancement` label.
+`critical` | Added to `bug` issues if the problem described makes the code completely unusable in a common situation. |
```

Markdownlint (0.37.0)

Expected: no_leading_or_trailing; Actual: trailing_only; Unexpected trailing pipe
Table pipe style

(MD055, table-pipe-style)